di_website/publications/models.py

```diff
@@ -137,20 +137,40 @@ def __str__(self):
         return self.country.name
 
 
+@register_snippet
+class ResourceCategory(ClusterableModel):
+    name = models.CharField(max_length=255, unique=True)
+
+    panels = [
+        FieldPanel('name'),
+    ]
+
+    class Meta:
+        ordering = ["name"]
+        verbose_name = 'Resource Categories'
```

Comment:
`Resource Category`
`verbose_name_plural = 'Resource Categories'`
Otherwise:
![image](https://user-images.githubusercontent.com/5672438/80567044-00b7d400-89fd-11ea-8d06-06d508ca4c9c.png)

Reply:
Thanks, this was a thorn